DigitalLearningSolutions.Data/Services/LoginService.cs

```diff
@@ -139,11 +135,18 @@ public LoginResult AttemptLogin(string username, string password)
             AdminUser? verifiedAdminUser
         )
         {
-            var verifiedLinkedAdmin = verifiedAdminUser ??
-                                      userVerificationService.GetVerifiedAdminUserAssociatedWithDelegateUsers(
-                                          approvedVerifiedDelegates,
-                                          password
-                                      );
+            var verifiedAssociatedAdmin = userVerificationService.GetVerifiedAdminUserAssociatedWithDelegateUsers(
+                approvedVerifiedDelegates,
+                password
+            );
+
+            if (approvedVerifiedDelegates.All(du => du.CentreId != verifiedAssociatedAdmin?.CentreId))

```

Comment:
Can we have a comment here explaining why we discard the admin we've found in this case?